lib/internal/modules/esm/load.js

```diff
@@ -131,11 +131,6 @@ async function defaultLoad(url, context = kEmptyObject) {
 
   validateAttributes(url, format, importAttributes);
 
-  // Use the synchronous commonjs translator which can deal with cycles.
-  if (format === 'commonjs' && getOptionValue('--experimental-require-module')) {
-    format = 'commonjs-sync';
-  }
-
```

Comment:
I don't really understand why this change is needed for unflagging 🤔

Reply:
It was previously affecting the async path which was unnecessary.